database/database.go

```diff
@@ -42,10 +42,8 @@ var (
 	// and it already exists there.
 	ErrSkylinkExists = errors.New("skylink already exists")
 
-	// Portal is the preferred portal to use, e.g. https://siasky.net
-	Portal string
-	// ServerDomain is the unique server name, e.g. eu-pol-4.siasky.net
-	ServerDomain string
+	// ServerUID is a random 16-char string that uniquely identifies the server
```

Comment:
```suggestion
	// ServerUID is a random string that uniquely identifies the server
```

doesn't have to be 16-char